_pages/soap_api/ccs_field_def_v1_5_service_attribute.md

```diff
@@ -23,6 +23,7 @@ The following table lists all Service Attributes currenty active in DoS.
 |requirebooking	| Boolean |This attribute identifies that the service can only be referred into with an appointment.|If this attribute is set to TRUE the DoS entry can be selected but it is not possible to make a referral. | When adding this attribute to a service it is important to note that not all 3rd party systems are using the attribute to inform system behaviour.  Therefore it is necessary to continue to maintain the referral notes on the service to indicate that referral is by appointment only. **_Note:_** _NHS Digital are working with the other suppliers as part of the development to support UEC Care Connect for booking and this will be a requirement.  See ([Booking only](https://developer.nhs.uk/apis/uec-appointments/dos_bookingonly.html)) page for more information about appointment booking._|
 |Digital_Go_With_Notif	| Boolean |This attribute indicates that a patient can physically go to the service and the service will also be notified of their referral. |If this attribute is set to TRUE the patient referral is expected to trigger an electronic notification. | When adding this attribute to a service it is important to note that the notification endpoint details must be profiled and correct.|
 |webchat_enabled | Boolean | This attribute is used by 111 Online as part of a webchat pilot service. | Where this attribute is linked to a service and is TRUE, 111 Online is able to offer webchat facility to patients. | To be used only with agreement from 111 Online and the participating service |
+|Website Signposting | Boolean |This attribute indicates that a service uses their website as their primary method of referral. |Where this attribute is linked to a service and is TRUE, 111 Online may direct the user to the listed website for the service. | To be used only with agreement from 111 Online and the participating service |
```

Comment:
Should there be a space in Website Signposting?  Not even an underscore? I guess it doesn't matter.